README.rst

```diff
@@ -37,6 +37,36 @@ Scrapy
    :target: https://anaconda.org/conda-forge/scrapy
    :alt: Conda Version
 
+Table of Contents:
```

Comment:
The ToC for documentation is situated on the documentation page. We shouldn't duplicate parts of it in README when we already link the documentation itself.